src/consumer.rs

```diff
@@ -11,6 +11,7 @@ use datafusion::{
     scalar::ScalarValue,
 };
 
+use datafusion_sql::TableReference;
```

Comment:
You can import from the main datafusion crate because it re-exports the SQL crate. We should not need to add a new dependency.

```suggestion
use datafusion::sql::TableReference;
```